quize.ipynb

```diff
@@ -0,0 +1,80 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "91ce22ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!/usr/bin/env python\n",
+    "# coding: utf-8\n",
+    "\n",
+    "# In[1]:\n",
+    "\n",
+    "\n",
+    "import csv \n",
+    "\n",
+    "\n",
+    "# In[4]:\n",
+    "\n",
+    "\n",
+    "#field name\n",
+    "fields=['name','contect']\n",
+    "data=[['om','6355786899'],['sai','9023137700'],\n",
+    "      ['ram','8160876821'],['kishan','9979246401']\n",
+    "     ]\n",
+    "filename=\"contect.csv\"\n",
+    "\n",
+    "\n",
+    "# In[6]:\n",
+    "\n",
+    "\n",
+    "with open(filename, 'w',newline='') as csvfile: \n",
+    "    csvwriter = csv.writer(csvfile) \n",
+    "    csvwriter.writerow(fields) \n",
```

Comment:
explain writerow() and writerows() function ?

Reply:
writerow() function writes items in a sequence (list, tuple or string) separating them by comma character.

writerows() function writes each sequence in a list as a comma separated line of items in the file.